inference.py

```diff
@@ -19,10 +19,10 @@
 import json
 import tarfile
 import time
-
+import sys
```

Comment:
Is this import needed?

Reply:
Thank you for pointing out. No, importing sys is not necessary and I will remove it. There is one thing I would like to mention.
With the PR #71, I am able to run inference and generate .csv prediction on cloud. However, output tar file cannot be generated on cloud, because tarfile package cannot open files in GCS. I tried several hacks (e.g. passing FileIO instead), but none of them seem to work.